File: 04-labs/2026-09-07-lab3-crash-workshop/solution/03_name.R
# 03 -- SOLUTION: the object is called `exports`, not `export`.
trade   <- read.csv("data/raw/comtrade_fr_roundwood_clean.csv")
exports <- subset(trade, flowDesc == "Export")
cat("Mean exported quantity:", mean(exports$qty, na.rm = TRUE), "m3\n")

File: 04-labs/2026-09-07-lab3-crash-workshop/solution/03_name.py
# 03 -- SOLUTION: the object is called `exports`, not `export`.
import pandas as pd
trade   = pd.read_csv("data/raw/comtrade_fr_roundwood_clean.csv")
exports = trade[trade.flowDesc == "Export"]
print("Mean exported quantity:", exports.qty.mean(), "m3")
